src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/DockerDecoratedLauncher.java

```diff
@@ -37,7 +37,11 @@ public Proc launch(ProcStarter starter) throws IOException {
         // Do not decorate launcher until SCM checkout completed
         if (!runInContainer.isEnabled()) return super.launch(starter);
 
-        runInContainer.getDocker().executeIn(runInContainer.container, starter);
+        try {
+            runInContainer.getDocker().executeIn(runInContainer.container, starter);
+        } catch (InterruptedException e) {
+            throw new IOException("Caught InterruptedException", e);
```

Comment:
InterruptedException is supposed to interrupt the build with a single click. BTW there's no chance to properly propagate the issue here due to the API.